assets/css/utilities/_hideable.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 .u-hide-for-mobile {
-  @media screen and (max-width: map-get($breakpoints, "max-mobile-landscape-tablet-portrait-width")) {
+  @include media-breakpoint-down(md) {
```

Comment:
@mbta/skate-developers want to surface this change. It's not a huge change, but this changes the "hide for mobile" breakpoint from 800px to 768px. 

I did this because `media-breakpoint-...` uses bootstrap's breakpoints, and I wanted to bias towards bootstrap's breakpoints rather than our own `_skate_ui` ones, especially since we want to stop "caring about what's in _skate_ui", per this [Slack Convo](https://mbta.slack.com/archives/C047ZT8ACP5/p1725383093148209?thread_ts=1725382780.112419&cid=C047ZT8ACP5).

If you'd all prefer, I can split it out into a separate PR. Or ditch this and find a different way to keep our breakpoints consistent!

Reply:
Btw I found the variable we need to update if we do want to make bootstrap use our breakpoints as a temporary measure https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L483-L492

Reply:
@firestack Amazing, thank you!

What do you think of [this](https://github.com/mbta/skate/pull/2774/files#diff-7781a17abc4cbb8e10f3131a4da4fcc8dc20efd270178f5b756e6be32ae32cafR20-R31)?

And at this point, should the `_variable_overrides` change and this one be split to a separate PR, do you think?

Reply:
> And at this point, should the `_variable_overrides` change and this one be split to a separate PR, do you think?

I'm good either way 👍🏻 


> What do you think of [this](https://github.com/mbta/skate/pull/2774/files#diff-7781a17abc4cbb8e10f3131a4da4fcc8dc20efd270178f5b756e6be32ae32cafR20-R31)?

Looks good, I don't _think_ you need `!default`? Did you try it without that? AFAIK bootstrap has that in their config so that we can override it easily.

Maybe it also should document _why_ these values are set like this, because right now we're doing this to be compatible with the old breakpoints "system", but I don't know what design will want to do with breakpoints in the future.

Reply:
[Added a comment](https://github.com/mbta/skate/pull/2774/files#diff-7781a17abc4cbb8e10f3131a4da4fcc8dc20efd270178f5b756e6be32ae32cafR20-R24)

I think I do want to split this out into two PR's.